docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,696 @@
+    // Unified commerce module
+    unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+      apiUrl: `${config.apiUrl}/commerce/unified`,
+      defaultRequestConfig: {
+        getConfigSwitcherHeader,
+        headers: getRequestHeaders(),
+      },
+      // ... other config
+    }),
+    // ... other modules
+  };
+});
+```
+::
+
+## Updating the File Structure for Path Routing
+
+To enable path-based routing, you need to restructure your application to include a dynamic `[configId]` parameter in the route. This allows you to access the store identifier directly from the URL.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+For Next.js apps using the App Router, update your file structure:
+
+```bash
+apps/storefront-unified-nextjs/app/
+├── [locale]/                 # Language parameter
+│   ├── [configId]/           # Store identifier parameter
+│   │   ├── (default)/        # Routes for all stores
+│   │   │   ├── page.tsx      # Home page
+│   │   │   ├── cart/         # Cart pages
+│   │   │   ├── checkout/     # Checkout pages
+│   │   │   └── products/     # Product pages
+│   │   ├── (electronics)/    # Electronics-specific routes
+│   │   │   └── some-page/    # Page name
+│   │   ├── (apparel)/        # Apparel-specific routes
+│   │   │   └── another-page/ # Apparel-specific home
+│   │   └── layout.tsx        # Apply store-specific class
+```
+
+#tab-2
+For Nuxt apps, update your file structure:
+
+```bash
+apps/storefront-unified-nuxt/pages/
+├── [configId]/              # Store identifier parameter
+│   ├── index.vue            # Home page
+│   ├── cart/                # Cart pages
+│   │   └── index.vue
+│   ├── checkout/            # Checkout pages
+│   │   └── index.vue
+│   ├── products/            # Product pages
+│   │   ├── index.vue
+│   │   └── [id].vue
+│   ├── some-electronics-page.vue # Electronics-specific page
+│   └── some-apparel-page.vue     # Apparel-specific page
+```
+::
+
+This structure enables routes like:
+- `/en/electronics` - Electronics store home
+- `/en/apparel` - Apparel store home
+- `/en/electronics/category` - PLP in the electronics store
+- `/en/apparel/category` - PLP in the apparel store
+
+## Updating Internal Links
+
+::warning Update Internal Links
+After setting up the path routing structure, you must update all internal links in your application to include the appropriate configId parameter. This is crucial to maintain proper navigation within each store and prevent users from accidentally switching between stores when clicking links.
+::
+
+Make sure that all your internal links and navigation components preserve the store context by including the current configId in the URLs. This applies to:
+
+- Navigation menus
+- Product links
+- Category links
+- Cart and checkout flows
+- Footer links
+- Breadcrumbs
+- Any other internal navigation elements
+
+Without this, users might unexpectedly switch between stores while browsing your site.
+
+## Applying Store-Specific Styling
```

Comment:
Group this with CSS section as it is too conceptually scattered for the reader.

Reply:
done https://github.com/vuestorefront/vue-storefront/pull/7422/commits/c8e0ca6d5af95e822a75a0d9dcc736f604b215f3